addons/event/views/event_registration_views.xml

```diff
@@ -132,44 +133,15 @@
         <field name="model">event.registration</field>
         <field name="priority">10</field>
         <field name="arch" type="xml">
-            <kanban class="o_event_attendee_kanban_view" default_order="name, create_date desc" sample="1">
+            <kanban class="o_event_attendee_kanban_view" default_order="name, create_date desc" sample="1" js_class="registration_summary_dialog_kanban">
                 <field name="name"/>
                 <field name="partner_id"/>
                 <field name="state"/>
                 <field name="email"/>
                 <field name="event_ticket_id"/>
                 <field name="active"/>
+                <field name="barcode"/>
                 <templates>
-                    <t t-name="event_attendees_kanban_icons_desktop">
-                        <div class="d-none d-md-block h-100">
-                            <div id="event_attendees_kanban_icons_desktop" class="h-100 float-end p-2 d-flex align-items-end flex-column gap-1">
-                                <t t-if="record.active.raw_value">
-                                    <a class="btn btn-md btn-secondary" string="Confirm Attendance" name="action_set_done" type="object" invisible="state == 'done'"
-                                        role="button" aria-label="Confirm Attendance Button" title="Confirm Attendance">
-                                        <i class="fa fa-check" role="img"/>
-                                    </a>
-                                    <a class="btn btn-md btn-success" string="Reset To Registered" name="action_confirm" type="object" invisible="state != 'done'"
-                                        role="button" aria-label="Reset To Registered Button" title="Reset To Registered">
-                                        <i class="fa fa-check" role="img"/>
-                                    </a>
-                                </t>
-                            </div>
-                        </div>
-                    </t>
-                    <t t-name="event_attendees_kanban_icons_mobile">
```

Comment:
Would keep the content, as people may use it in custom code. Just remove it in master.